buildSrc/src/main/kotlin/io/spine/internal/gradle/ProjectExtensions.kt

```diff
@@ -72,16 +73,14 @@ fun <T : Task> Project.findTask(name: String): T {
 }
 
 /**
- * Obtains the Maven artifact ID of the project taking into account
- * the value of the [PublishExtension.spinePrefix] property.
+ * Obtains the Maven artifact ID of the project.
  *
- * If the project has a [PublishExtension] installed, then the extension is used for
- * [obtaining][PublishExtension.artifactId] the artifact ID.
- *
- * Otherwise, the project name is returned.
+ * Uses `mavenJava` publication by convention.
```

Comment:
Did you mean this instead?
```
Uses `mavenJava` conventions for publication.
```

I don't really understand the original sentence. And how it relates to `artifactId`. Let's discuss vocally if needed.